elcid/templates/records/icu_round.html

```diff
@@ -14,10 +10,22 @@
     [[ item.fio2 ]]
    <br />
 </span>
-<span ng-show="item.inotropic_dose">
+<span ng-show="item.inotrope_dose">
     [[ item.inotropic_dose ]]
    <br />
 </span>
+<span ng-show="item.inotrop">
+    [[ item.inotropic_drug ]]
+   <br />
+</span>
+<span ng-show="item.vasopressor_dose">
+    [[ item.inotropic_dose ]]
```

Comment:
I think this should be `vasopressor_dose`

Reply:
Sure - not even the only typo in this file.
It's never rendered, so reasonably meh.